staging/src/k8s.io/apiserver/pkg/storage/watch_cache_test.go

```diff
@@ -93,37 +115,42 @@ func TestWatchCacheBasic(t *testing.T) {
 	}
 
 	// Test List.
-	store.Add(makeTestPod("pod1", 4))
```

Comment:
Why this name change? Would prefer minimal changes to the test

Reply:
This was to keep the names of the pods and variables consistent (`pod1` is already defined in this test). Reverted.